client/src/pages/Home.css

```diff
@@ -1,63 +1,8 @@
-:root {
-  --small-icon-width:35px;
-  --medium-icon-width:33px;
-  --large-icon-width:30px;
-}
-
 body {
   background-color: #FDF6F0;
 }
 
-.buttons {
-  padding: 5px;
-  background-color: #FCD8D4;
-  display: flex;
-  justify-content: space-between;
-}
-
 .homepage-header {
   background-color: #FCD8D4;
   padding: 5px;
 }
```

Comment:
Sorry for the additional nitpick - could you move this to `HomePageHeader.css` as well, even though it wasn't a duplicate? In order to keep our code clean it's best to write styles in the file associated with the component they belong to.

Reply:
Hi, Joe. I will remove the `<Header className=homepage-header />` and the associated css since it does not show anything after we replaced it with `<HomePageHeader />`

Reply:
Perfect, thank you!